com.regnosys.rosetta.tests/src/com/regnosys/rosetta/generator/java/function/FunctionGeneratorTest.xtend

```diff
@@ -562,6 +563,7 @@ class FunctionGeneratorTest {
 		code.compileToClasses
 	}
 
+	@Disabled
```

Comment:
Test contains type errors (`exists` on required attribute). I don't get what this test is supposed to verify, so I have disabled it. Might just remove it?